pkg/controller/s3/s3bucket.go

```diff
@@ -210,7 +208,7 @@ func updateTagging(bucket *bucketv1alpha3.S3Bucket, client s3.Service) error {
 
 func (r *Reconciler) _delete(bucket *bucketv1alpha3.S3Bucket, client s3.Service) (reconcile.Result, error) {
 	bucket.Status.SetConditions(runtimev1alpha1.Deleting(), runtimev1alpha1.ReconcileSuccess())
-	if bucket.Spec.ReclaimPolicy == runtimev1alpha1.ReclaimDelete {
+	if bucket.Spec.DeletionPolicy == runtimev1alpha1.DeletionDelete {
```

Comment:
iirc Delete is now the default. Do we need to handle the case where the policy not set?

Reply:
This controller will be [gone](https://github.com/crossplane/provider-aws/pull/331) in the next days, so I don't think we need to handle it. I just made changes to get the controller in a state that can be compiled.